opentreemap/treemap/models.py

```diff
@@ -1230,6 +1230,7 @@ class Boundary(models.Model):
                                       db_index=True)
 
     canopy_percent = models.FloatField(null=True)
+    searchable = models.BooleanField(default=True)
```

Comment:
We like to keep our migrations backwards compatible by using `NullBooleanField` and then making it non-nullable in a follow up migration.